ansible_collections/arista/avd/molecule/eos_designs_unit_tests/intended/structured_configs/cv-pathfinder-edge.yml

```diff
@@ -142,8 +142,53 @@ router_bfd:
     min_rx: 300
     multiplier: 3
 router_path_selection:
+  path_groups:
+  - name: INET
+    id: 500
+    local_interfaces:
+    - name: Ethernet1
+      stun:
+        server_profiles:
+        - cv-pathfinder-pathfinder-INET-0
+    dynamic_peers:
+      enabled: true
+    static_peers:
+    - router_ip: 192.168.44.1
+      name: cv-pathfinder-pathfinder
+      ipv4_addresses:
+      - 10.7.7.7
+    ipsec_profile: CP-PROFILE
+  - name: MPLS
+    id: 500
```

Comment:
Will need a different ID here else the config will fail.

Reply:
Good catch - made ID mandatory and added some future task!